_rules/document-has-no-repeated-content-g0m9ub.md

```diff
@@ -0,0 +1,184 @@
+rule_type: atomic
+description: |
+  This rule checks that each page has no repeated content before its main content.
+accessibility_requirements:
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation
+
+The test target contains no [block of repeated content][] which is located before its [main block of content][]
+
+## Assumptions
+
+- This rule assumes that there is exactly one [main block of content][] inside each [HTML web page][].
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+[Success Criterion 2.4.1 Bypass Blocks][sc241] require a way to "bypass blocks of content that are repeated on multiple Web pages." However, [Understanding Success Criterion 2.4.1: Bypass Blocks][usc241] states that the intent is to allow "more direct access to the primary content of the Web page" and many of its sufficient techniques only provide a way to skip to the main content (most notably [G1: Adding a link at the top of each page that goes directly to the main content area][tech g1]). It is not fully clear whether there must be a way to skip repeated content located after the primary content (literal reading of the Success Criterion suggest so); or to skip non-repeated content located before the primary content (such as local navigation) (the Understanding Document and sufficient techniques suggest so). In any case when this rule fails, it is a clear case where means to bypass blocks must be provided in order to satisfy [Success Criterion 2.4.1 Bypass Blocks][sc241].
+
+Because the definitions of [block of repeated content][] and [main block of content][] are subjective (they rely on interpreting the purpose of some content), they cannot be used in the Applicability of any rule. This rule is essentially a substitute for an Applicability of [Bypass Blocks of Content][] and its other atomic rules. Only when this rule fails is there a clear need to provide a way for bypassing content located before the [main block of content][].
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+In this document, the [main block of content][] is the `main` element. The `aside` element is a [block of repeated content][] because it is also present in the document linked at the end of the `main` element. There is no [block of repeated content][] before the [main block of content][]. The only [block of repeated content][] is located after the [main block of content][]. Note that the `nav` element is not a [block of repeated content][].
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<nav>
+			<a href="#main">Skip to main content</a>
```

Comment:
Good question… The intention was to show that there can be **something** before the main content, as long as it is not repeated. 
* In Passed Example 3, I guess the skiplink makes sense because the `aside` could still be repeated on other pages of the site (and thus the skiplink is needed to pass the SC), but our restrictive definition of "repeated" wouldn't detect that (e.g., the website could have an entry page with links to each Chapter, and then the individual chapter pages would have no link, the `aside` would still be repeated even though we don't detect it with our definition).
* In Passed Example 1 and 2, the skiplink indeed seems weird…